docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+# Docker compose configuration file for the HMDA Platform
+# Version: 1.0.0
+
+hmda:
+  build: .
+  ports:
+    - 8080:8080
+    - 2551:2551
```

Comment:
The default port for an ActorSystem. Will come into play shortly, although it's not being used now I wanted to include it in the configuration